app/src/main/res/values/styles.xml

```diff
@@ -29,8 +29,10 @@
         <item name="android:paddingEnd">16dp</item>
         <item name="android:textColor">@drawable/primary_button_text</item>
         <item name="android:textSize">16sp</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">sans-serif</item>
+        <item name="android:textStyle">bold</item>
         <item name="textAllCaps">false</item>
+        <item name="android:lineSpacingExtra">@dimen/default_line_spacing</item>
```

Comment:
This one should be definitely be moved as well or maybe removed. Why do you need line spacing in a button? Buttons should have 1 line of text and in the worst case scenario ellipse. Maybe use padding to adjust to match the mocks but please also do so in the button in that particular view.

Reply:
I just took this from the Zeplin layout. You are probably right.

Reply:
Fixed in cf62d47e2a713d1b7b9de2f41c71e15e0cd55eda